packages/select/docs/FilterMultiSelect.stories.tsx

```diff
@@ -72,10 +72,10 @@ export const DefaultKaizenSiteDemo: ComponentStory<
           </>
         )}
       </FilterMultiSelect>
-      <Paragraph variant={"body"}>
-        Items:{" "}
+      <div style={{ marginTop: 4 }}>
+        <Paragraph variant="body">Items: </Paragraph>{" "}
```

Comment:
```suggestion
        <Paragraph variant="body">Items: </Paragraph>
```

Doesn't look like you need it (based on the other section) :)